pkgs/tools/networking/firefoxpwa/default.nix

```diff
@@ -0,0 +1,100 @@
+{ stdenv
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, symlinkJoin
+, buildFHSUserEnv
```

Comment:
I think ```buildFHSUserEnv``` has been renamed ```buildFHSEnv```.

Also it seems like the FHS env script fails since the latest updates for me: the Firefox window does not open and there is no error. If I use the unwrapped binary, it works!